.github/build_scripts/setup.py

```diff
@@ -0,0 +1,58 @@
+import os
+
```

Comment:
we are almost there!! 

why a separate setup.py file?

Reply:
In order not to affect other build options. For example, building from source will not work correctly if an extension is added to the setup.py. It also allows us keeps everything we need for build in one place and it's makes workflow a little cleaner.

Reply:
isn't there an environment field we can check for so we know we are building a binary wheel? I don't like the idea of code duplication.